src/Table/util/breakpoint.ts

```diff
@@ -0,0 +1,42 @@
+import type { ColMinBreakpoint } from "..";
+
+const breakpointOrder = ["s", "m", "l"] as const;
+type OrderedBreakpoint = (typeof breakpointOrder)[number];
+
+/**
+ * We use "mobile-first" breakpoints, meaning that
+ * "m" is `true` for medium viewports and larger.
+ * (e.g. "l" satisfies "m")
+ */
+export const isBreakpointSatisfied = (
```

Comment:
I gave this some thought, and I feel that we should move this to the `useBreakpoints` hook.

We'd add two items to the returned object in `useBreakpoints`:

1. `isBreakpointSatisfied(breakpointName)` - compare a bp name to current
2. `currentBreakpoint` - name of current or closest breakpiont